### PR TITLE
Graphics

### PR DESCRIPTION
**Pull request by dominikjasek**
Graphics should be working now.

you need to install _pygame_ to run this

use this command in cmd
`pip install pygame`

### CODE REVIEW

File: Graphics/colors.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Colors:
+  def __init__(self):
+    self.BG_DARK = (155,205,69)
```
Comment (pilkaj) [inline Graphics/colors.py]:
```
suggestion, for more precise naming:
    self.BG_FIELD_DARK = (155,205,69)
```

File: Graphics/colors.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    self.BG_LIGHT = (170,215,81)
```
Comment (pilkaj) [inline Graphics/colors.py]: as above

File: Graphics/colors.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+    self.SNAKE_HEAD = (200,0,0)
+    self.SNAKE_TAIL = (0,0,200)
```
Comment (pilkaj) [inline Graphics/colors.py]:
```
suggestion: (since tail might be only last joint of snake)
    self.SNAKE_BODY = (0,0,200)
```

**Review by pilkaj — APPROVED** [review]

File: Graphics/colors.py (hunk 1 above)
Comment (dominikjasek) [reply Graphics/colors.py]: I agree, will rename

File: Graphics/colors.py (hunk 3 above)
Comment (dominikjasek) [reply Graphics/colors.py]: Resolved. Thanks.

**Pull request merged**